Projects/Checkers/Program.cs

```diff
@@ -0,0 +1,227 @@
+﻿Encoding encoding = Console.OutputEncoding;
+
+try
+{
+	Console.OutputEncoding = Encoding.UTF8;
+	Game game = ShowIntroScreenAndGetOption();
+	Console.Clear();
+	RunGameLoop(game);
+	RenderGameState(game, promptPressKey: true);
+	Console.ReadKey(true);
+}
```

Comment:
do you want to catch any exceptions for logging or to throw a human readable message to user here?

Reply:
Nope. I want to let exceptions fly. The goal of this code is to be an educational example for other developers. Yes logging is an important part of coding, but in this case I don't want to suppress any exceptions. If people modify the code as an educational exercise I want them to see exceptions they cause.